modmesh/app/euler1d.py

```diff
@@ -317,42 +292,6 @@ def editable(self, row, col):
             return True
         return False
```

Comment:
Since `PlotConfig` table has multiple columns allowing user to modified it's value via `QtTableView`. Therefore I overwrite the `editable` method.